File: orderservice/src/main/java/com/orderservice/dto/CartItemDTO.java
package com.orderservice.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.*;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class CartItemDTO {
    @JsonProperty("product_id")
    private Long productId;

    @JsonProperty("quantity")
    private Integer quantity;

    private Float price;
}

File: frontEnd/src/app/dto/cart/cartItem.dto.ts
export class CartItemDTO {    
    product_id: number;
    
    quantity: number;

    price: number;
    
      
    constructor(data: any) {
        this.product_id = data.product_id;
        this.quantity = data.quantity;    
        this.price = data.price;
    }
}